FlightReservation/src/consts/custom-theme.json

```diff
@@ -0,0 +1,77 @@
+  "color-primary-transparent-300": "rgba(23, 161, 162, 0.24)",
+  "color-primary-transparent-400": "rgba(23, 161, 162, 0.32)",
+  "color-primary-transparent-500": "rgba(23, 161, 162, 0.4)",
+  "color-primary-transparent-600": "rgba(23, 161, 162, 0.48)",
+  "color-success-100": "#FEF0D0",
+  "color-success-200": "#FEDCA2",
+  "color-success-300": "#FEC373",
+  "color-success-400": "#FDAB50",
+  "color-success-500": "#FC8417",
+  "color-success-600": "#D86510",
+  "color-success-700": "#B54A0B",
+  "color-success-800": "#923307",
+  "color-success-900": "#782304",
+  "color-success-transparent-100": "rgba(252, 132, 23, 0.08)",
+  "color-success-transparent-200": "rgba(252, 132, 23, 0.16)",
+  "color-success-transparent-300": "rgba(252, 132, 23, 0.24)",
+  "color-success-transparent-400": "rgba(252, 132, 23, 0.32)",
+  "color-success-transparent-500": "rgba(252, 132, 23, 0.4)",
+  "color-success-transparent-600": "rgba(252, 132, 23, 0.48)",
+  "color-info-100": "#D4E6FE",
+  "color-info-200": "#AACBFE",
+  "color-info-300": "#80ADFC",
+  "color-info-400": "#5F93FA",
+  "color-info-500": "#2C69F7",
+  "color-info-600": "#2050D4",
+  "color-info-700": "#163BB1",
+  "color-info-800": "#0E288F",
+  "color-info-900": "#081B76",
+  "color-info-transparent-100": "rgba(44, 105, 247, 0.08)",
+  "color-info-transparent-200": "rgba(44, 105, 247, 0.16)",
+  "color-info-transparent-300": "rgba(44, 105, 247, 0.24)",
+  "color-info-transparent-400": "rgba(44, 105, 247, 0.32)",
+  "color-info-transparent-500": "rgba(44, 105, 247, 0.4)",
+  "color-info-transparent-600": "rgba(44, 105, 247, 0.48)",
+  "color-warning-100": "#FEF6CC",
+  "color-warning-200": "#FEEB9A",
+  "color-warning-300": "#FCDC67",
+  "color-warning-400": "#FACD41",
+  "color-warning-500": "#F7B604",
+  "color-warning-600": "#D49602",
+  "color-warning-700": "#B17902",
+  "color-warning-800": "#8F5D01",
+  "color-warning-900": "#764A00",
+  "color-warning-transparent-100": "rgba(247, 182, 4, 0.08)",
+  "color-warning-transparent-200": "rgba(247, 182, 4, 0.16)",
+  "color-warning-transparent-300": "rgba(247, 182, 4, 0.24)",
+  "color-warning-transparent-400": "rgba(247, 182, 4, 0.32)",
+  "color-warning-transparent-500": "rgba(247, 182, 4, 0.4)",
+  "color-warning-transparent-600": "rgba(247, 182, 4, 0.48)",
+  "color-danger-100": "#FCE6D3",
+  "color-danger-200": "#FAC8A9",
+  "color-danger-300": "#F19F7C",
+  "color-danger-400": "#E47859",
+  "color-danger-500": "#D34128",
+  "color-danger-600": "#B5271D",
+  "color-danger-700": "#971415",
+  "color-danger-800": "#7A0C16",
+  "color-danger-900": "#650717",
+  "color-danger-transparent-100": "rgba(211, 65, 40, 0.08)",
+  "color-danger-transparent-200": "rgba(211, 65, 40, 0.16)",
+  "color-danger-transparent-300": "rgba(211, 65, 40, 0.24)",
+  "color-danger-transparent-400": "rgba(211, 65, 40, 0.32)",
+  "color-danger-transparent-500": "rgba(211, 65, 40, 0.4)",
+  "color-danger-transparent-600": "rgba(211, 65, 40, 0.48)"
+}
```

Comment:
Do you need all this range of colors?

Reply:
It was a theme for a framework but we decided not to use it, so we already deleted that file.